### PR TITLE
Added homework

### PR DESCRIPTION
**Pull request by nare99**
1. Added "Create package for modules" homework(package dir)
2. Added sort homework(sort dir)

### CODE REVIEW

File: Nare_Mkrtchyan/python/26_05/sort/sort.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+while i != 0:
+    user_input = input()
+    user_input = user_input.strip()
+    if check(user_input):
+        i = i - 1
+    
+letter()
+number()
+print("\nList after sorting:")
+print(list1)
```
Comment (ani-im) [inline Nare_Mkrtchyan/python/26_05/sort/sort.py:153]: valid input ->
image

Comment (ani-im) [reply Nare_Mkrtchyan/python/26_05/sort/sort.py:153]: Incorrect result
image

File: Nare_Mkrtchyan/python/26_05/sort/sort.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+print("Structure of string: a12 asd dfg    or   f12 45 456")
+i = list_len
+while i != 0:
+    user_input = input()
+    user_input = user_input.strip()
+    if check(user_input):
+        i = i - 1
+    
+letter()
+number()
```
Comment (nare99) [inline Nare_Mkrtchyan/python/26_05/sort/sort.py:151]: asd

Comment (ani-im) [reply Nare_Mkrtchyan/python/26_05/sort/sort.py:151]: asd

**Review by ani-im — CHANGES_REQUESTED** [review]

File: Nare_Mkrtchyan/python/26_05/sort/sort.py (hunk 1 above)
Comment (nare99) [reply Nare_Mkrtchyan/python/26_05/sort/sort.py:153]: Corrected founded mistakes.

**Pull request merged**